getstarted.md

```diff
@@ -22,6 +22,70 @@ with Hadoop, as well as extensions to that foundation that make for totally
 awesome demos. You'll also find a few videos related to our work and of course,
 how to contact us!
 
+First, though, let's just take a look at how straightforward it is to get started with very complex software. You can do it during a lunch break:
```

Comment:
I don't mind the immediate intro into a simple "lunch break" demo.  I think it's way better than (as pete said) throwing the users a bunch of charms and hoping for the best.